release_benchmarks.sh

```diff
@@ -34,3 +31,11 @@ data+=("$RESULT")
 
 # Save to data.json
 printf "%s\n" "${data[@]}" > $OUTPUT_FILE
+
+cd docs
+
+git config user.name "Travis CI"
+git config user.email "github@fb.com"
```

Comment:
Why does this use an old fb bot?

Reply:
I don’t know the traviscibot email, I’ll try to change it.

Reply:
Or I could just leave my face on there

Reply:
I would think its fine if it's your face, or maybe we could create a new parse community bot account

Reply:
Do you know what website theme is used for docs and blog?

Reply:
not sure

Reply:
For the website it says -